### DISCUSSION TITLE
Found under repo's actions

### DISCUSSION THREAD

**Discussion by Universalizer** [xarantolus/filtrite#24] [Show and tell]
Hello @xarantolus
Thanks for creating one of this greatest repository and it's good setup. 

For past few weeks, noticing under repo's actions, as follows

Annotations
1 warning
[Build](https://github.com/Universalizer/Universaliser_Filters/actions/runs/4679983334/jobs/8290737419)
Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16: actions/setup-go@v2, actions/checkout@v2, dev-drprasad/delete-older-releases@v0.1.0, actions/upload-artifact@v2. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/.Show less

**Comment by xarantolus** [comment]
Thank you for pointing this out :)
